src/Evolution/Systems/Cce/Actions/CMakeLists.txt

```diff
@@ -21,6 +21,7 @@ spectre_target_headers(
   InitializeFirstHypersurface.hpp
```

Comment:
Can you make the commit message more descriptive? Maybe `Add action computing Psi0 at the worldtube`?